pubspec.lock

```diff
@@ -28,7 +28,7 @@ packages:
       name: async
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "2.6.1"
+    version: "2.5.0"
```

Comment:
I see some packages are downgraded again, did you run `flutter pub get` before your commit?

Reply:
I think the versions are not matter, it should work fine if we stay with this version.

Reply:
If there is no specific reason to downgrade a package, we should not do that. By downgrading a version we may even loose functionality in external libraries and by such break functionality without noticing. Please revert your changes to the pubspec.lock and run `flutter pub get` to install the latest packages.